src/main/java/uk/gov/hmcts/reform/professionalapi/authchecker/JwtGrantedAuthoritiesConverter.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.hmcts.reform.professionalapi.authchecker;
+
```

Comment:
can we include all the files added for oidc in package oauth rather than using different packages  for different files?